docs/manual/docs/cmd/spo/apppage/apppage-update.md

```diff
@@ -0,0 +1,29 @@
+# spo apppage update
+
+Updates the single-part app page
+
+## Usage
+
+```sh
+spo apppage update [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|The URL of the site where the page to update is located
+`-p, --pageName <pageName>`|The name of the page to be updated, eg. page.aspx
```

Comment:
the short for `--pageName` says `-n` instead of `-p`. I guess we can leave with `-p` as well, it is up to Waldek to decide.

Reply:
`, eg. page.aspx` is missing from the command option description. Could you please add it there as well?

Reply:
Let's use `-n` so that it's in sync with other commands where we have page name, eg. `spo page get`